pkgs/applications/science/misc/root/default.nix

```diff
@@ -114,8 +114,8 @@ stdenv.mkDerivation rec {
     python.pkgs.numpy
     tbb
   ]
-  ++ lib.optionals (!stdenv.isDarwin) [ libX11 libXpm libXft libXext libGLU libGL ]
-  ++ lib.optionals (stdenv.isDarwin) [ Cocoa CoreSymbolication OpenGL ]
+  ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [ libX11 libXpm libXft libXext libGLU libGL ]
+  ++ lib.optionals (stdenv.hostPlatform.isDarwin) [ Cocoa CoreSymbolication OpenGL ]
```

Comment:
What is this for? hostPlatform is the default lookup, and it's not like we are going to implement cross compilation.

Reply:
> hostPlatform is the default lookup

Glad to know that. I'll drop that commit.